.npmignore

```diff
@@ -16,3 +16,4 @@ npm-debug.log
 node_modules
 coverage
 coverage-e2e
+test/
```

Comment:
Not in this case. All files [should _end_ with a new line](https://robots.thoughtbot.com/no-newline-at-end-of-file).